recipe/patch_yaml/libdeflate.yaml

```
@@ -0,0 +1,76 @@
+# The libdeflate ABI is very stable. Loosen pins to reflect this.
+#
+# Once a new libdeflate minor release is determined to be ABI compatible, the
+# `new:` fields below need to be updated to new: libdeflate>=1.3,<1.X.0a0, where
+# X is the new minor release.
+#
+# This is the strategy used by the bioconda channel
+# https://github.com/bioconda/bioconda-recipes/blob/2b170ae75dcc4b2a17c53fd4b77554e75178c2d3/recipes/bioconda-repodata-patches/gen_patch_json.py#L174
+#
+if:
+  has_depends: libdeflate*
+  timestamp_lt: 1742582718000
+then:
+  - replace_depends:
+      old: libdeflate >=1.23,<1.24.0a0
+      new: libdeflate >=1.3,<1.24.0a0
+  - replace_depends:
+      old: libdeflate >=1.22,<1.23.0a0
+      new: libdeflate >=1.3,<1.24.0a0
+  - replace_depends:
+      old: libdeflate >=1.21,<1.22.0a0
+      new: libdeflate >=1.3,<1.24.0a0
+  - replace_depends:
+      old: libdeflate >=1.20,<1.21.0a0
+      new: libdeflate >=1.3,<1.24.0a0
+  - replace_depends:
+      old: libdeflate >=1.19,<1.20.0a0
+      new: libdeflate >=1.3,<1.24.0a0
+  - replace_depends:
+      old: libdeflate >=1.18,<1.19.0a0
+      new: libdeflate >=1.3,<1.24.0a0
```

If anything, you can only lift the upper bound, but never the lower one.
```suggestion
  - replace_depends:
      old: libdeflate >=1.18,<1.19.0a0
      new: libdeflate >=1.18,<1.24.0a0
```

---

Good point! That was my mistake. I had misinterpreted the effect of the bioconda patching code. In retrospect it makes sense. I've fixed the lower bounds